app/uk/gov/hmrc/bindingtariffclassification/utils/JsonUtil.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.utils
+
+import play.api.libs.json.{Format, OFormat, JsObject, JsValue, JsResult}
+
+object JsonUtil {
+
+  def convertToOFormat[T](format: Format[T]): OFormat[T] = {
```

Comment:
I'd maybe just make this a wrapper around `JsonX` so we can call `JsonUtil.format[SomeType]` and thats it

Reply:
Could even have two
`JsonUtil.formatUsingDefaults[SomeType]` which delegates to JsonX
`JsonUtil.format[SomeType]` which delegates to Play Json (`Json.format[T]`)